### ISSUE TITLE
Transitions not working with minified CSS

### ISSUE THREAD

**Issue by mweiczorek** [Semantic-Org/Semantic-UI#6739]
Hi,

Transitions are not working for minified CSS, but do work if the un-minified file is used.

My setup is the latest version of Senatntic (2.4.2) and semantic-ui-react (0.84.0). The project is an Electon/React/TypeScript application, so it's not an incredible deal using an un-minified CSS file, but thought I'd share this issue in case this affects users who need to use the minified file in a web-base project.

### Steps

Standard install using Gulp, no other alterations made from default except file output locations in my semantic.json file.

I've included a Gist of both generated files, both minified and not.

[Generated Semantic-UI CSS Gist](https://gist.github.com/mweiczorek/f114f573da55e8731fbea684ea8f548c)

**Comment by hammy2899** [comment]
@mweiczorek Are you using webpack with react?

**Comment by mweiczorek** [comment]
No, I use FuseBox, and it never touches the CSS. The generated CSS is referenced from the index.html file that houses the entire application, and is never touched.
